src/grc65/main.c

```diff
@@ -850,8 +850,12 @@ static char *filterInput (FILE *F, char *tbl)
     /* loads file into buffer filtering it out */
     int a, prevchar = -1, i = 0, bracket = 0, quote = 1;
 
-    for (;;) {
-        a = getc(F);
+    a = getc(F);
+    while (1)
+    {
+        if (i >= BLOODY_BIG_BUFFER) {
+            AbEnd ("File too large for internal parsing buffer (%d bytes).",BLOODY_BIG_BUFFER);
+        }
         if ((a == '\n') || (a == '\015')) a = ' ';
         if (a == ',' && quote) a = ' ';
         if (a == '\042') quote =! quote;
```

Comment:
While at it, would you add braces here too, please?

Reply:
Indeed, you could also combine both space assignments.

Reply:
I'm unclear about what this means. Where's the missing braces in my change?

Reply:
Oh, you're reviewing the old pre-existing code. Sorry, that was confusing to me.

Reply:
[532681c](https://github.com/cc65/cc65/pull/2098/commits/532681c9613af15b1993e08db12761855c9707c6)